src/components/Card/CardVariants.ts

```diff
@@ -0,0 +1,136 @@
+import { ExternalStyles, focusBoxShadow, hexToRGB, Theme } from '../../styles'
+
+export type CardVariant = 'outline' | 'float' | 'plain'
+
+export const createBaseStyles = (theme: Theme) => ({
+  card: {
+    width: '100%',
+    border: 0,
+    borderRadius: `${theme.radius.popper}px`,
+    padding: '1rem',
+    transition: 'all .2s',
+  } as ExternalStyles,
+  cardDisabled: {
+    cursor: 'not-allowed',
+    '& *': {
+      pointerEvents: 'none',
+    },
+  } as ExternalStyles,
+  innerDisabled: {
+    opacity: 0.4,
+  } as ExternalStyles,
+})
+
+export const createVariantStyles = (theme: Theme): { [key in CardVariant]: ExternalStyles } => ({
```

Comment:
```suggestion
export const createVariantStyles = (theme: Theme): Record<key in CardVariant, {base, disabled, invalid}> => ({
```
Then you don't need to use the `data-` attributes

Reply:
done :heavy_check_mark: